mtags-interfaces/src/main/java/scala/meta/pc/PresentationCompiler.java

```diff
@@ -266,6 +266,8 @@ public abstract PresentationCompiler withScheduledExecutorService(
 	 */
 	public abstract PresentationCompiler withWorkspace(Path workspace);
 
+	public abstract PresentationCompiler withReferenceCounter(ReferenceCountProvider provider);
```

Comment:
We also use old mtags for scala versions that aren't supported anymore, so each new added method needs a default implementation. This is the reason for most of the CI fail.